plugins/chat/spec/system/channel_members_page_spec.rb

```diff
@@ -31,11 +31,12 @@
     end
 
     context "with members" do
+      fab!(:searched_user) { Fabricate(:user) }
+
       before do
         channel_1.add(current_user)
-        channel_1.add(Fabricate(:user, username: "cat"))
```

Comment:
setting username or not is unrelated with flakeyness, it would fail when removed too